app/src/main/res/layout/fragment_breathless.xml

```diff
@@ -76,22 +76,33 @@
             android:layout_marginStart="40dp"
             android:layout_marginBottom="30dp"
             android:onClick="@{() -> vm.onClickSkip()}"
+            isClickable="@{vm.isInProgress}"
             android:text="@string/symptoms_skip_button"
             android:textSize="18sp"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@id/cause_recycler_view" />
 
-        <ProgressBar
-            android:id="@+id/progress"
-            style="@style/screen_progress_indicator"
+
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
I'd put this in an include (reusable XML) or a custom view (which may be needed to block touch). It was ok before to repeat the progress bar's XML, because it was minimal. This is too much repeated code.